examples/Planning/ForecastReach.cs

```diff
@@ -0,0 +1,313 @@
+﻿// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Ads.GoogleAds.Lib;
+using Google.Ads.GoogleAds.V2.Errors;
+using Google.Ads.GoogleAds.V2.Common;
+using Google.Ads.GoogleAds.V2.Services;
+
+using System;
+using System.Collections.Generic;
+using static Google.Ads.GoogleAds.V2.Enums.DeviceEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.GenderTypeEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.ReachPlanAdLengthEnum.Types;
+using static Google.Ads.GoogleAds.V2.Enums.ReachPlanAgeRangeEnum.Types;
+
+namespace Google.Ads.GoogleAds.Examples.V2
+{
+    /// <summary>
+    /// This code demonstrates how to generate a video ads reach forecast.
+    /// </summary>
+    public class ForecastReach : ExampleBase
+    {
+        /// <summary>
+        /// Main method, to run this code example as a standalone application.
+        /// </summary>
+        /// <param name="args">The command line arguments.</param>
+        public static void Main(string[] args)
+        {
+            ForecastReach codeExample = new ForecastReach();
+            Console.WriteLine(codeExample.Description);
+
+            // The customer ID for which the call is made.
+            int customerId = int.Parse("INSERT_CUSTOMER_ID_HERE");
+
+            codeExample.Run(new GoogleAdsClient(), customerId);
+        }
+
+        /// <summary>
+        /// Returns a description about the code example.
+        /// </summary>
+        public override string Description
+        {
+            get
+            {
+                return "This code example generates a reach forecast for a given targeting and preferences.";
```

Comment:
The code example docstring doesn't match this description. Please fix.

Reply:
Updated to make it clear the code example is for video ads.